src/uniprotkb/components/entry/Entry.tsx

```diff
@@ -99,26 +89,50 @@ const Entry: React.FC<EntryProps> = ({ addMessage, match }) => {
     [transformedData]
   );
 
+  const listOfIsoformAccessions = useMemo(
```

Comment:
Rewrote a bit this logic in the latest commit. Didn't do reduce because it messes up with TypeScript types and I think it night be less complex to read with just for loops.